config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'queues/spam'
```

Comment:
Try to keep routes in a `scope` where possible, to group them together logically. These could all go in `scope '/queues`.

Reply:
Whoops! Those were autogenerated.